src/strict.lua

```diff
@@ -1,4 +1,3 @@
---!nonstrict
```

Comment:
can you try changing these !nonstrict removals to !strict? if they're not strict-clean, leave the !nonstrict so folks who want to strictify checking on their app+packages don't get noise they can't do something about. (If they're impossible to make clean due to Luau issue, also add a `-- FIXME Luau`.